tutorials/en_us/Advanced JAR picking.md

```diff
@@ -1,70 +1,51 @@
-If you want to go a little more in-depth with JAR picking, you've come to the right place.
-Most popular Spigot/Bukkit JARs are :  
-1. PaperSpigot
-
-2. Spigot
-
-3. CraftBukkit (rarely used)                  These support using Spigot/Bukkit plugins, don't try mods with them (won't work)
+# Advanced JAR picking
 
-4. Purpur
-
-5. Tuinity
-#
+If you want to go a little more in-depth with JAR picking, you've come to the right place.
+Most popular Spigot/Bukkit JARs are :
 
-1. PaperSpigot - Most popular JAR for Spigot/Bukkit, quite fast and stable, they roll out small patches at least every 2 days or so, good choice if you're just
-                 starting with your MC server and don't have much past experience because it's fast and simple.
+1. [PaperMC](https://papermc.io/) - Most popular JAR for Spigot/Bukkit, quite fast and stable, they roll out small patches at least every 2 days or so, good choice if you're just
+   starting with your MC server and don't have much past experience because it's fast and simple.
+1. [Spigot](https://www.spigotmc.org/) - Original fork of Bukkit, improved performance vs CraftBukkit, still widely used, seems to be harder for newbies to download because it links to Jenkins,
```

Comment:
Why use a numbered list here?
Also they probably shouldn't all be `1.`
Imo change to unordered list - using `-` dashes.